worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,34 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	int lives = 5;
+	bool playerHasChosen;
+	std::string playerInput;
+	std::string guess;
+	int wordLikeness;
+
+	while (lives > 0)
+	{
+		playerHasChosen = false;
+		while (playerHasChosen == false)
+		{
+			std::cout << "Enter your guess in Capitals\n";
+			std::cin >> playerInput;
+			playerHasChosen = true;
```

Comment:
The flowchart specifies that the program should check the guessed word is present in the list of options. Was that what the `while` loop was for maybe, and you forgot the check?